docs(incremental_backups): explain NBD (#7240)
vatesfr/xen-orchestra: docs/incremental_backups.md

This is important because on rare occasions a backup can be corrupted, and in the case of incremetnal backups, this corruption might impact all the following backups in the chain. Occasionally performing a full backup limits how far a corrupted delta backup can propagate.

The value to use depends on your storage constraints and the frequency of your backups, but a value of 20 is a good start.

## NBD-enabled Backups

You have the option to utilize the NBD network protocol for data transfer instead of the VHD handler generated by the XAPI. NBD-enabled backups generally show improved speed as the load on the Dom0 is reduced.

To enable NBD on the pool network, select the relevant pool, and navigate to the Network tab to modify the parameter:

![](./assets/nbdconnection.png)

> 

Reviewer: ```suggestion

This will transfer data encrypted from the host to the XOA. 
```

> When creating or editing a delta backup on this pool in the future, you can activate NBD under the Advanced settings:

Reviewer: we're trying to transition to the wording incremental backup , so maybe we can use 
```suggestion
When creating or editing a incremental (formerly delta) backup and replication on this pool in the future, you can activate NBD under the Advanced settings:
```

![](./assets/enablenbd.png)

After the job is completed, you can verify whether NBD was used for the transfer in the backup log:

![](./assets/backuplog.png)

> It's important to note that NBD exports are not currently visible in the task list, but we are actively addressing this. Stay tuned for more updates in upcoming versions!

Reviewer: this should be merged during this release : https://github.com/vatesfr/xen-orchestra/pull/7228 , so  :
```suggestion
It's important to note that NBD exports are not currently visible in the task list before 5.90 (december 2023).
```

This is important because on rare occasions a backup can be corrupted, and in the case of incremetnal backups, this corruption might impact all the following backups in the chain. Occasionally performing a full backup limits how far a corrupted delta backup can propagate.

The value to use depends on your storage constraints and the frequency of your backups, but a value of 20 is a good start.

## NBD-enabled Backups

You have the option to utilize the NBD network protocol for data transfer instead of the VHD handler generated by the XAPI. NBD-enabled backups generally show improved speed as the load on the Dom0 is reduced.

To enable NBD on the pool network, select the relevant pool, and navigate to the Network tab to modify the parameter:

![](./assets/nbdconnection.png)

> This will transfer data encrypted from the host to the XOA. 

Author: ```suggestion
This will securely transfer encrypted data from the host to the XOA.
```

> When creating or editing a incremental (formerly delta) backup and replication on this pool in the future, you can activate NBD under the Advanced settings:

Author: ```suggestion
When creating or editing an incremental (previously known as delta) backup and replication for this pool in the future, you have the option to enable NBD in the Advanced settings:
```
